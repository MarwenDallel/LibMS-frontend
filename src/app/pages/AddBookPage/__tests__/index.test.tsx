import * as React from 'react';
import { render } from '@testing-library/react';
import { createMemoryHistory } from 'history';
import { AddBookPage } from '..';
import { Router } from 'react-router-dom';

const renderWithRouter = (component: JSX.Element) => {
  const history = createMemoryHistory();
  return {
    ...render(<Router history={history}>{component}</Router>),
  };
};

describe('<AddBookPage  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = renderWithRouter(<AddBookPage />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
