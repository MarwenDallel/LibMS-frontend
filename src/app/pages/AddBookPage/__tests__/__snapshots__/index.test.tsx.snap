// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddBookPage  /> should match snapshot 1`] = `
.c1 {
  background-color: #007ea4;
}

.c0 {
  background-color: #203549;
}

<div
  class=""
>
  <nav
    class="c0 navbar navbar-expand navbar-light"
  >
    <div
      class="container"
    >
      <a
        href="/"
      >
        <img
          alt=""
          height="122"
          src="SMU-LOGO.png"
          width="263"
        />
      </a>
    </div>
  </nav>
  <nav
    class="c1 navbar navbar-expand-md navbar-dark navbar navbar-expand-lg navbar-light"
  >
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="justify-content-center justify-content-md-between container"
      >
        <div
          class="font-weight-light mr-0 text-white"
        >
          Book Adding Form
        </div>
      </div>
    </div>
  </nav>
</div>
`;
